content/guides/box-mcp/self-hosted/index.md

```diff
@@ -0,0 +1,460 @@
+
+### Use Claude for Desktop as the Box MCP client
+
+Prerequisites:
+
+- Download [Claude for Desktop](https://claude.ai/download) client
+- Optionally, set up [`code`](https://code.visualstudio.com/docs/setup/mac#_configure-the-path-with-vs-code) command for VS Code
+
+Follow this steps to set up Box MCP in Claude for Desktop:
+
+1. Edit your `claude_desktop_config.json`.
+
+To do so you can run this command in your terminal:
+
+```sh
+code ~/Library/Application\ Support/Claude/claude_desktop_config.json
+```
+
+Alternatively, in main Claude navigation choose `Settings`. Select the Developers tab and click `Edit Config`. This will open a folder window containing `claude_desktop_config.json`.
+
+2. Add the configuration:
+
+```json
+{
+  "mcpServers": {
+      "mcp-server-box": {
+          "command": "uv",
+          "args": [
+              "--directory",
+              "/Users/Users/USER_NAME/PATH_TO_PROJECT/mcp-server-box",
+              "run",
+              "src/mcp_server_box.py"
+          ]
+      }
+  }
+}
+```
+
+3. You might need to restart Claude for Desktop.
+4. Authenticate Box MCP Server with `box_authorize_app_tool` tool.
+
+## Available tools
+
+### Authentication and user tools
+
+| Tool | Description | Parameters | Returns |
+|------|-------------|------------|---------|
+| `box_who_am_i` | Get current user information and check connection status | None | User information string |
+| `box_authorize_app_tool` | Start the Box application authorization process | None | Authorization status message |
+
+### Search and navigation tools
+
+<table>
+  <thead>
+    <tr>
+      <th>Tool</th>
+      <th>Description</th>
+      <th>Parameters</th>
+      <th>Returns</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>`box_search_tool`</td>
+      <td>Search for files in Box</td>
+      <td>
+        - `query (str):` Search query
+        - `file_extensions (List[str], optional):` Filter by extensions
+        - `where_to_look_for_query (List[str], optional):` Locations to search
+        - `ancestor_folder_ids (List[str], optional):` Folder IDs to limit the search
+      </td>
+      <td>Newline-separated list of file names and IDs</td>
+    </tr>
+    <tr>
+      <td>`box_search_folder_by_name`</code></td>
+      <td>Locate a folder by name</td>
+      <td>`folder_name (str):` Name of the folder</td>
+      <td>Folder ID and information</td>
+    </tr>
+    <tr>
+      <td>`box_list_folder_content_by_folder_id`</td>
+      <td>List folder contents</td>
+      <td>
+          <ul>
+             <li>`folder_id (str):` ID of the folder</li>
+             <li>`is_recursive (bool):` Whether to list recursively</li>
+          </ul>
+       </td>
+      <td>Folder content in JSON format</td>
+    </tr>
+  </tbody>
+</table>
+
+### File operations
+
+<table>
+  <thead>
+    <tr>
+      <th>Tool</th>
+      <th>Description</th>
+      <th>Parameters</th>
+      <th>Returns</th>
```

Comment:
I think you can add another column to the table: `Type`
You can mark the required params with an asterisk, as it is commonly used in such context. Add a short explanation before or after the table, for example: `* required params` or `params marked with a * are required`.

Reply:
Good point, I'll try to refine this tomorrow